IBPSA/Fluid/Chillers/BaseClasses/Carnot.mo

```diff
@@ -24,8 +24,7 @@ partial model Carnot
   parameter Boolean use_eta_Carnot_nominal = true
     "Set to true to use Carnot effectiveness etaCarnot_nominal rather than COP_nominal"
     annotation(Dialog(group="Efficiency"));
-  parameter Real etaCarnot_nominal(unit="1") = COP_nominal/
-    (TUseAct_nominal/(TCon_nominal+TAppCon_nominal - (TEva_nominal-TAppEva_nominal)))
+  parameter Real etaCarnot_nominal(unit="1") = 0.3
```

Comment:
Should we remove the value from the examples, as it does not alter the default?

Reply:
@mwetter : Please check this command. The other parameters with a default are not overwritten in the examples, e.g. here:
![image](https://user-images.githubusercontent.com/22442514/217284978-313936ac-3f0d-47d1-a600-52567f14d808.png)
This would be in line with keeping the defaults in the example:
![image](https://user-images.githubusercontent.com/22442514/217285068-c407ea80-bb61-4a00-a9b7-90091520af5f.png)